apis/swagger.yml

```diff
@@ -1086,6 +1086,36 @@ paths:
             $ref: "#/responses/500ErrorResponse"
         tags: ["Network"]
 
+  /networks/{id}/connect:
+    post:
+      summary: "Connect a container to a network"
+      operationId: "NetworkConnect"
+      consumes:
+        - "application/json"
+      responses:
+        200:
+          description: "No error"
+        404:
```

Comment:
Please add 400, since you have code `return httputils.NewHTTPError(err, http.StatusBadRequest)`